docs/.vitepress/theme/index.js

```diff
@@ -1,5 +1,6 @@
 import DefaultTheme from 'vitepress/dist/client/theme-default'
 import Layout from './Layout.vue'
+import './styles/index.css'
```

Comment:
I pushed a change in CSS for this, you can remove it. It will avoid adding `div`s

Reply:
Great. I deleted this css file